packages/navigation/src/hooks/useWebviewNavigate.ts

```diff
@@ -105,6 +105,9 @@ function getMinimalAction(
   while (
     currentAction.payload &&
```

Comment:
We still need to compare the params of the screen to ensure that we do not remove screens which have same name, but different params as discussed on Slack